lib/command_runner.dart

```diff
@@ -400,13 +420,15 @@ String _getCommandUsage(Map<String, Command> commands,
   var buffer =
       new StringBuffer('Available ${isSubcommand ? "sub" : ""}commands:');
   for (var name in names) {
-    var lines = commands[name].summary.split("\n");
+    var columnStart = length + 5;
```

Comment:
Nit: This can be hoisted out of the loop.

I would tend to do this not because it helps performance (it probably doesn't) but to make it more visually clear that the value of this variable doesn't depend on `name`.